rmk-config/src/keymap.pest

```diff
@@ -98,12 +98,15 @@ shifted_action = { ^"SHIFTED" ~ "(" ~ keycode_name ~ ")" }
 // Rule 8: TD(n) - Tap Dance
 tap_dance_action = { ^"TD" ~ "(" ~ number ~ ")" }
 
+// Rule 9: M(n) - Trigger Macro
```

Comment:
I think users might be confused by `M(n)` and `MO(n)`, they looks similar but they're totally different. Use `Macro(n)` maybe?

Reply:
Yes, rename it please